@navikt/core/css/config/_mappings.js

```diff
@@ -178,6 +178,11 @@ const StyleMappings = {
     { component: "Tag", main: "tag.css", dependencies: [typoCss] },
     { component: "TextField", main: formCss, dependencies: [typoCss] },
     { component: "Textarea", main: formCss, dependencies: [typoCss] },
+    {
+      component: "FormSummary",
+      main: formCss,
+      dependencies: ["form-summary.css", typoCss],
```

Comment:
Siden form-summary.css er en del av "form"-scopet trenger den ikke å være en dependency
```suggestion
      dependencies: [typoCss],
```